src/plugins/wb-randomize/wb-randomize.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @title WET-BOEW Randomize
+ * @overview This plugin randomly picks one of the child component to be shown on the browser.
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @masterbee
+ */
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "wb-randomize",
+	selector = ".provisional[data-wb-randomize]",
+	initEvent = "wb-init" + selector,
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			$elm = $( elm ),
+			setting = $elm.data( "wb-randomize" ),
+			selected = $( setting.selector, $( elm ) );
+
```

Comment:
Here add a conditional that selected must be valid and contains items.
Throw an error if it is not provided or return nothing